test_integrated_query.py::test_two_simultaneous_runs fails

Error occurs during docker execution:

```
/workdir/rel/source/analysis/Root/query.cxx: In member function 'virtual StatusCode query::execute()':
/workdir/rel/source/analysis/Root/query.cxx:100:19: warning: unused variable 'i_obj12' [-Wunused-variable]
         for (auto i_obj12 : *tracks10)
                   ^~~~~~~
/workdir/rel/source/analysis/Root/query.cxx:76:15: warning: unused variable 'i_obj9' [-Wunused-variable]
     for (auto i_obj9 : *jets8)
               ^~~~~~
Traceback (most recent call last):
  File "../source/analysis/share/ATestRun_eljob.py", line 58, in <module>
    driver.submit(job, options.submission_dir)
TypeError: none of the 2 overloaded methods succeeded. Full details:
  void EL::Driver::submit(const EL::Job& job, const string& location) =>
    failed to submit job (C++ exception of type runtime_error)
  void EL::Driver::submit(const EL::Job& job, const string& location, string& actualLocation) =>
    takes at least 3 arguments (2 given)
```

--- a/tests/atlas/xaod/test_integrated_query.py
+++ b/tests/atlas/xaod/test_integrated_query.py
@@ -50,7 +50,7 @@ def event_loop():
 async def test_two_simultaneous_runs():
     # Test the future return stuff
     f_training_df_1 = as_pandas_async(f_single
-                                      .Select('lambda e: e.Jets("AntiKt4EMTopJets").Select(lambda j: e.Tracks("InDetTrackParticles")).First().Count()'))
+                                      .Select('lambda e: e.Jets("AntiKt4EMTopoJets").Select(lambda j: e.Tracks("InDetTrackParticles")).First().Count()'))
     f_training_df_2 = as_pandas_async(f_single
                                       .Select('lambda e: e.Jets("AntiKt4EMTopoJets").Select(lambda j: e.Tracks("InDetTrackParticles")).First().Count()'))
     r1, r2 = await asyncio.gather(f_training_df_1, f_training_df_2)

